domain/secretbackend/state/state_test.go

```diff
@@ -132,7 +129,13 @@ func (s *stateSuite) createModelWithName(c *gc.C, modelType coremodel.ModelType,
 		userUUID,
 		// TODO (stickupkid): This should be AdminAccess, but we don't have
 		// a model to set the user as the owner of.
-		permission.ControllerForAccess(permission.SuperuserAccess),
+		permission.AccessSpec{
+			Access: permission.SuperuserAccess,
+			Target: permission.ID{
+				ObjectType: permission.Controller,
+				Key:        s.controllerUUID,
```

Comment:
Then s.ControllerUUID here.

Reply:
This stays as it is for the reason in the commend above.